kernel/src/actions/mod.rs

```diff
@@ -47,8 +48,8 @@ static LOG_SCHEMA: LazyLock<SchemaRef> = LazyLock::new(|| {
         Option::<Protocol>::get_struct_field(PROTOCOL_NAME),
         Option::<SetTransaction>::get_struct_field(SET_TRANSACTION_NAME),
         Option::<CommitInfo>::get_struct_field(COMMIT_INFO_NAME),
+        Option::<Cdc>::get_struct_field(CDC_NAME),
```

Comment:
Does this PR need to rebase? I could have sworn `Cdc` infra already merged?

Reply:
Yep, needs a rebase